test/api/ocpp/json/OCPPJsonService16.ts

```diff
@@ -145,10 +145,6 @@ export default class OCPPJsonService16 extends OCPPService {
   }
 
   private async send(chargeBoxIdentity: string, message: any): Promise<any> {
-    // Debug
-    // console.log('OCPP Request ====================================');
-    // console.log({ chargeBoxIdentity, message });
-    // console.log('====================================');
```

Comment:
Same comment, don't remove.

Reply:
Then add a tunable to conditionally enable debug code path.

Reply:
No